File: C programming/sum of digits.c
#include <stdio.h>

int main() {
    int num, remainder, sum = 0;

    printf("Enter a number: ");
    scanf("%d", &num);

    while (num != 0) {
        remainder = num % 10;
        sum += remainder;
        num /= 10;
    }

    printf("Sum of digits = %d\n", sum);
    return 0;
}

File: Java programming/Day-2/sum of digits.java
import java.util.Scanner;

public class SumOfDigits {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter an integer: ");
        int num = scanner.nextInt();
        
        int sum = 0;
        int temp = Math.abs(num); 
        
        while (temp > 0) {
            sum += temp % 10;
            temp /= 10;       
        }
        
        System.out.println("The sum of digits in " + num + " is: " + sum);
        scanner.close();
    }
}
